src/xrAICore/Navigation/graph_abstract.h

```diff
@@ -21,13 +21,13 @@ template <typename _data_type = Loki::EmptyType, typename _edge_weight_type = fl
 class CGraphAbstract
 {
 public:
-    typedef CVertex<_data_type, _vertex_id_type, CGraphAbstract> CVertex;
+    typedef CVertex<_data_type, _vertex_id_type, CGraphAbstract> CVrtx;
```

Comment:
CAbstractGraphVertex is probably better name than CVrtx

Reply:
A long name is worse because this typedef is used in other templates, and they are so poorly readable.

Reply:
Why type definition is named with `C` prefix? C stands for "class" in short name. Maybe it should be named with `T` prefix? (Type) or if it confusing with Template we can call it just `GraphVrx` since it's just typedef member of  `CGraphAbstract`. What you think about that?

Reply:
`vertex_t`?

Reply:
If we change the code that contains `typedef`, we can replace it with `using`. That will be more expressive